engine/api/src/main/java/io/deephaven/engine/table/Table.java

```diff
@@ -368,7 +368,8 @@ public interface Table extends
      * <p>
      * Returns a subset of table in the range [startPercentInclusive * sizeOfTable , endPercentExclusive * sizeOfTable).
```

Comment:
I suggest including the floors so that super quant nerds and number lawyers know what is going on.